docs/tutorials/kubernetes/efficient-data-sharing-in-kubeflow-with-vineyard-csi-driver.rst

```diff
@@ -14,6 +14,7 @@ Prerequisites
 guide `Initialize Kubernetes Cluster`_ to create one.
 - Install the `Vineyardctl`_ by following the official guide.
 - Install the `Argo Workflow CLI`_ by following the official guide.
+- Install the kfp package with version < 2.0.0.
```

Comment:
Why we requires <2.0.0 here?

Reply:
The current demo is based on the kfp v1.8.0. In kfp v2, we need to rewrite the demo with the new API.

Reply:
As the kfp v2 has some breaking changes compared to v1. Backgound:  https://www.kubeflow.org/docs/components/pipelines/v2/migration/

Actually, I think it doesn't matter as they are complied as the argo workflow YAML finally.